packages/common/test/location/location_spec.ts

```diff
@@ -113,5 +113,31 @@ describe('Location Class', () => {
          expect((location as any)._urlChangeListeners.length).toBe(1);
          expect((location as any)._urlChangeListeners[0]).toEqual(changeListener);
        }));
+
+    it('should only notify listeners once when multiple listeners are registered',
+       inject(
+           [Location, LocationStrategy],
+           (location: Location, locationStrategy: MockLocationStrategy) => {
+             let notificationCount = 0;
```

Comment:
```suggestion
       () => {
       const location = TestBed.get(Location);
       const locationStrategy = TestBed.get(MockLocationStrategy);
             let notificationCount = 0;
```
nit: can we update this to use the newer `TestBed` APIs?

Reply:
I'm trying to follow the code style of this test suite. If we want to use `TestBed`, shouldn't we change all the test cases? Also, we should use `TestBed.inject`.

Reply:
Right, use `TestBed.inject` instead. Also, same comment as above - please still update this one to the newer style and then follow up with a PR to update refactor the others. Thanks!